numojo/core/datatypes.mojo

```diff
@@ -1,15 +1,15 @@
 """
 # ===----------------------------------------------------------------------=== #
 # Datatypes Module - Implements datatypes aliases, conversions
-# Last updated: 2024-06-17
+# Last updated: 2024-06-18
 # ===----------------------------------------------------------------------=== #
 """
 
 
-alias int8 = DType.int8
-alias int16 = DType.int16
-alias int32 = DType.int32
-alias int64 = DType.int64
+alias it8 = DType.int8
```

Comment:
If we are going to abbreviate the word integer smaller than int I think `i32` us more recognizable as an int than `it32`

Reply:
I made Rust-like alias for data types in the following pull request to shivasankarka's branch.

https://github.com/shivasankarka/NuMojo/pull/7

Reply:
I like the "rust" like naming better. We can stick to that I guess.